lambda/aws-qa-appsync-opensearch/question_answering/src/Dockerfile

```diff
@@ -1,17 +1,9 @@
-FROM public.ecr.aws/lambda/python:3.11.2023.11.13.10-x86_64
+FROM public.ecr.aws/lambda/python:3.12
```

Comment:
How might multi-architecture builds be tested (`arm` vs `x86_64` and platforms too)?

Reply:
That is a good point. Do you have any recommendations on how to do that?

Reply:
Good point. I don't have much experience in this arena. Do you have any recommendations?

Reply:
The building platform is by default chosen, so when deploying `linux/amd64` from macOS there are occasional python packages downloaded that are incompatible - not a blocker, but something to look out.